package.json

```diff
@@ -3,7 +3,10 @@
   "id": "firefox-voice@mozilla.org",
   "description": "This is a WebExtension that allows the browser to be voice-controlled.",
   "private": true,
-  "version": "0.19.0",
+  "engines": {
+    "node": ">=12.16.1"
+  },
+  "engineStrict": true,
```

Comment:
Generally **engineStrict** is used if we have clear idea about the least version to use. **12.16.1** is a bit high version for that.

According me engineStrict should be avoided in our case.

@ianb am i right?

Reply:
Probably we can just say `>=12.0.0` – at least unless we test it out and are sure that there's something in a later version that we require. 12.16.1 is very exact, so it _seems_ like we're being specific, but I'm assuming this just happens to be the latest 12 release.

I wasn't familiar with engineStrict, but looking it up it seems the property is deprecated: https://docs.npmjs.com/files/package.json#enginestrict